deploy/helm-chart/kubernetes-replicator/values.yaml

```diff
@@ -1,26 +1,37 @@
-# Default values for kubernetes-replicator.
-# This is a YAML-formatted file.
-# Declare variables to be passed into your templates.
+replicaCount: 1
 
 image:
   repository: quay.io/mittwald/kubernetes-replicator
-  tag: stable
-  pullPolicy: IfNotPresent
+  #tag: stable # if no tag is given, the chart's appVersion is used
```

Comment:
If we don't set a tag anymore, don't we need to update the chart's `appVersion` with each deployment? Not to say that's a bad idea, but I didn't find the code for that anywhere. Or is that done by the mittwald/helm-charts repo workflow?

Reply:
For now, the chart's `appVersion` has to be increased manually... but getting that done automagically on tags is a great idea. I'll give this a shot, gimme a sec :3

Reply:
>For now, the chart's appVersion has to be increased manually

You KNOW that you yourself would be the first one to forget this, dont't you? 😉

Reply:
Hmm not sure... I haven't changed anything on the replicator itself, therefore there will be no new release. Instead I've updated the `version` of the chart.